.github/workflows/build-wheels-windows-dispatch_new.yml

```diff
@@ -0,0 +1,33 @@
+name: windows-dispatch
+
+on:
+  pull_request:
```

Comment:
How is this definition supposed to work? If you set it up this way, it will be triggered on every push to any branch of any PR to this repository. 

Is that how it is intended?

Reply:
Thank you.
No it is not, it's just for now to trigger the workflow check.
I should leave a comment here or we can merge into master right version with schedule.